Assets/SEE/Controls/DesktopPlayerMovement.cs

```diff
@@ -22,11 +23,28 @@ private struct CameraState
 
         private CameraState cameraState;
 
+        /// <summary>
+        /// Unity component that moves the player constrained by collisions.
+        /// It moves the player with its own <see cref="CharacterController.Move(Vector3)"/> method.
+        /// </summary>
+        private CharacterController controller;
+
         [Tooltip("The code city which the player is focusing on.")]
         public GO.Plane FocusedObject;
 
         private void Start()
         {
+            controller = gameObject.MustGetComponent<CharacterController>();
+
+            // Default Layer is ignored by collider
+            controller.excludeLayers = 1;
```

Comment:
The default layer is 0. The value 1 is for the built-in TransparentFX layer. Is this comment correct?

Is there any way to avoid this magic number?

Reply:
That was also something what confused me, when i looked up the Layers in Unity-Editor. There is a way _LayerMask.NameToLayer("Default")_ to get the Index by Name. But there is also something strange, the int is 0 in this case but when I checked the inspector it choosed the "Nothing"-Layer. If I use LayerMask.NameToLayer("Default")+1, so its 1, it shows the "Default"-Layer in inspector.

Reply:
This would be worth mentioning in the code as a comment.

Reply:
I think i found a better way for this magic number now. Seems like LayerMask is working as a BitMask (See commit #550ca76 for this)

Reply:
The problem is fixed now.